pkg/kf/apps/zz_generated.client.go

```diff
@@ -321,3 +327,62 @@ func (core *coreClient) Upsert(namespace string, newObj *v1alpha1.App, merge Mer
 
 	return core.Create(namespace, newObj)
 }
+
+// WaitFor is a convenience wrapper for WaitForE that fails if the error
+// passed is non-nil. It allows the use of Predicates instead of ConditionFuncE.
+func (core *coreClient) WaitFor(namespace string, name string, interval time.Duration, timeout *time.Duration, condition Predicate) (*v1alpha1.App, error) {
```

Comment:
Instead of a timeout, can we use a context? That would better suit more use cases.